Averages/projects/averages_project_3.py

```diff
@@ -3,104 +3,64 @@
 import numpy as np
 
 
-
-#calculate and print out the prediction based on ML
-def get_prediction(url, data={"A":48,"B":23,"C":38,"D":54}):
+#calculate and print out the prediction
+def get_prediction(url, flag, data={"A":48,"B":23,"C":38,"D":54}):
```

Comment:
The prints should be outside the function so that we dont change the prediction message based on the flag!

Reply:
Ah ok makes sense.